Resources/translations/role.en.xlf

```diff
@@ -1,212 +1,215 @@
 <?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
   <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
     <body>
-      <trans-unit id="b51b764f4a4a4454ce7ae9aa09052a93" resname="role.dashboard_title">
+      <trans-unit id="fbefa01af4762e0ece92415ab7d1783f" resname="role.dashboard_title">
```

Comment:
Mysteeeeerious :) Ok, I'll approve. Crossing fingers it won't somehow bite us later.